.travis.yml

```diff
@@ -53,6 +53,10 @@ addons:
       - opam
       - php-codesniffer
       - verilator
+      - jpegoptim
```

Comment:
the templates will now automatically generate this for the `precise` job.

Ideally, you also add it to *either* the hand coded xenial or trusty job that run in the first batch of jobs, in `.moban.dt/travis-manual-matrix.yaml`